src/main/java/org/cloudfoundry/promregator/scanner/ReactiveAppInstanceScanner.java

```diff
@@ -116,6 +133,18 @@ public String getAccessURL() {
 		 */
 		public void setAccessURL(String accessURL) {
 			this.accessURL = accessURL;
+    }
+    /**
+		 * @return the isInternal
+		 */
+		public boolean getIsInternal() {
```

Comment:
getters of beans of type boolean do not have the `get` prefix, but always are only `is` prefixed.